GherkinSyncTool.Synchronizers.AzureDevOps/AzureDevopsSynchronizer.cs

```diff
@@ -49,6 +49,12 @@ public void Sync(List<IFeatureFile> featureFiles)
 
             foreach (var featureFile in featureFiles)
             {
+                if (featureFile.Document.Feature is null)
```

Comment:
Can `Document `property be null?

Reply:
I've tested with an empty feature file, and only the Feature property was null.